framework/src/base/Assembly.C

```diff
@@ -2159,14 +2146,12 @@ Assembly::reinitElemFaceRef(const Elem * elem,
 }
 
 void
-Assembly::computeADFace(const Elem * elem, unsigned int side)
+Assembly::computeADFace(const Elem * elem, const unsigned int side)
```

Comment:
You changed the other method signature to a reference. Might as well do the same here. I believe this is only used internally in this class so shouldn't have to change any code outside of here. We dereference the pointer on the first line so a reference would be safer.

Reply:
Sounds good, works for me. I've been slowly cleaning up internal signatures like this as I see them